web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -852,6 +852,18 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
   @volatile var snippetNamesToSearch: FactoryMaker[String => List[String]] =
       new FactoryMaker(() => (name: String) => name :: Nil) {}
 
+
+  /**
+   * Snippet timers are used to time and record the execution time of snippets. We provide
+   * two default implementations for you:
+   *   - NoOpSnippetTimer that does nothing
+   *   - LoggingSnippetTimer that logs snippet times.
+   *
+   * Since this is a FactoryMaker you can programmatically override it for an individual request
+   * or session as you see fit. You can also implement your own timer!
+   */
+  val snippetTimer = new FactoryMaker[SnippetTimer](() => NoOpSnippetTimer){}
```

Comment:
Worth noting that FactoryMakers can have significant overhead. Tempted to say this is best done as a var that can perhaps hold an `Option[FactoryMaker]` or something of the sort.

Reply:
Ah, good call.